frontend/src/metabase/static-viz/components/WaterfallChart/constants.ts

```diff
@@ -0,0 +1,47 @@
+export const WATERFALL_CHART_TYPE = "waterfall";
+
+export const TIME_SERIES_WATERFALL_CHART_DEFAULT_OPTIONS = {
+  data: [
+    ["2020-10-20", 20],
+    ["2020-10-21", 20],
+    ["2020-10-22", 100],
+    ["2020-10-23", -10],
+    ["2020-10-24", 20],
+    ["2020-10-25", -30],
+    ["2020-10-26", -10],
+    ["2020-10-27", 20],
+    ["2020-10-28", -15],
+  ],
+  settings: {
+    show_values: true,
+  },
+  labels: {
+    left: "Count",
+    bottom: "Created At",
+  },
+  type: "timeseries",
+};
+
```

Comment:
Do all of the static viz components have default values like this? I don't know that it makes sense to have default data? I think things should fail if no data is passed in, rather than put default data in there.

Unless this is just for visual confirmation in an internal page?

Reply:
This is only used for rendering our internal page. i.e. this
https://github.com/metabase/metabase/blob/0fd17b376cb02a0337e5ae2c5bcd6cd65f5b1a8e/frontend/src/metabase/internal/pages/StaticVizPage.jsx#L157-L173
![image](https://user-images.githubusercontent.com/1937582/191928231-a27d24c1-a701-4a3b-9c8c-dd8957a6b28b.png)

@alxnddr What do you think about renaming the file to something else, maybe `dev-constants` to communicate this isn't supposed to or meant to be used in the actual rendering?

By the way, we don't have other names other than `constants.(ts|js)` should we just rename the constant names instead? Renaming it to something like `INTERNAL_PAGE_TIME_SERIES_WATERFALL_CHART_DEFAULT_OPTIONS` could make it clear, I, however, don't like the fact that `constants.ts` is still used for ilr internal page. What do you think?